supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -167,10 +167,25 @@ trait SharedFileSystemAsyncJobExecutionActor
     * Writes the script file containing the user's command from the WDL as well
     * as some extra shell code for monitoring jobs
     */
-  private def writeScript(instantiatedCommand: String, cwd: Path) = {
+  private def writeScript(instantiatedCommand: String, cwd: Path, globFiles: Set[WdlGlobFile]) = {
     val rcPath = if (isDockerRun) jobPaths.toDockerPath(jobPaths.returnCode) else jobPaths.returnCode
     val rcTmpPath = s"$rcPath.tmp"
 
+    def globManipulation(globFile: WdlGlobFile) = {
```

Comment:
This looks like a direct copy/paste? Maybe we can extract this into `backendEngineFunctions` too?